pkgs/development/python-modules/qdarkstyle/default.nix

```
@@ -0,0 +1,21 @@
+{ stdenv, fetchPypi, buildPythonPackage }:
+
+buildPythonPackage rec {
+  pname = "QDarkStyle";
```

```suggestion
  pname = "qdarkstyle";
```
The line should instead be put inside `fetchPypi`.